### PR TITLE
graphic design blogpost

### PR DESCRIPTION
**Pull request by shannonbux**
@calcsam ready!

### CODE REVIEW

File: docs/blog/2018-07-07-graphic-design-class/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: Learning how to code with Gatsby
+date: "2018-07-07"
+author: "Shannon Soper"
+tags: ["college", "case study"]
```
Comment (calcsam) [inline docs/blog/2018-07-07-graphic-design-class/index.md]: design, getting-started

File: docs/blog/2018-07-07-graphic-design-class/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+Recently, a Gatsby fan named Phil Howley starting teaching Gatsby in an Emerging Technologies class at the Art Institute (AI) in North Hollywood and I got to interview him.
+
+In his 10 years of teaching over 50 web development/design classes at (AI) (his other job is with [https://ironcovesolutions.com](https://ironcovesolutions.com) which is built on Gatsby!), he’s observed that graphic design students tend to come to class less as the class gets more focused on coding.
+
+However, in his current class _nobody has dropped out_. And every single one of his students turned in a portfolio, which is also unheard of in this particular class. The reason? He taught them to create their portfolios using Gatsby.
+
+## Here are some of the student’s repos and final sites:
+
+![Image of graphic designer's site](graphic-design-1.png)
```
Comment (calcsam) [inline docs/blog/2018-07-07-graphic-design-class/index.md]: Can we format these this way:

[Site Title](http://site-url) ([source](http://github-repo))

and also make the image link to the site?

**Comment by calcsam** [comment]
Looks great! left a couple of comments.

File: docs/blog/2018-07-07-graphic-design-class/index.md (hunk 1 above)
Comment (shannonbux) [reply docs/blog/2018-07-07-graphic-design-class/index.md]: do you mean add these to tags or replace current tags with these 2?

Comment (shannonbux) [reply docs/blog/2018-07-07-graphic-design-class/index.md]: I have no attachment to current ones, and don't know much about how we're using tags ;)

Comment (calcsam) [reply docs/blog/2018-07-07-graphic-design-class/index.md]: add these in :) take a look at `/blog/tags/`

Comment (shannonbux) [reply docs/blog/2018-07-07-graphic-design-class/index.md]: ok done! Thx!

**Pull request merged**